plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestClickHouseConnectorTest.java

```diff
@@ -352,120 +341,33 @@ protected TestTable createTableWithUnsupportedColumn()
-        // GROUP BY with JOIN
-        assertThat(query(joinPushdownEnabled(getSession()), "" +
-                "SELECT n.regionkey, sum(c.acctbal) acctbals " +
-                "FROM nation n " +
-                "LEFT JOIN customer c USING (nationkey) " +
-                "GROUP BY 1"))
-                .isNotFullyPushedDown(
-                        node(JoinNode.class,
-                                anyTree(node(TableScanNode.class)),
-                                anyTree(node(TableScanNode.class))));
-
-        // decimals
         String schemaName = getSession().getSchema().orElseThrow();
-        try (TestTable testTable = new TestTable(onRemoteDatabase(),
-                schemaName + ".test_aggregation_pushdown",
-                "(short_decimal decimal(9, 3), long_decimal decimal(30, 10)) Engine=Log",
-                ImmutableList.of("100.000, 100000000.000000000", "123.321, 123456789.987654321"))) {
-            assertThat(query("SELECT min(short_decimal), min(long_decimal) FROM " + testTable.getName())).isFullyPushedDown();
-            assertThat(query("SELECT max(short_decimal), max(long_decimal) FROM " + testTable.getName())).isFullyPushedDown();
-            assertThat(query("SELECT sum(short_decimal), sum(long_decimal) FROM " + testTable.getName())).isFullyPushedDown();
-            assertThat(query("SELECT avg(short_decimal), avg(long_decimal) FROM " + testTable.getName())).isFullyPushedDown();
+        try (TestTable testTable = createTableWithNumericTypes(schemaName + ".test_aggregation_pushdown",
+                ImmutableList.of("100.000, 100000000.000000000, 100000000, 100.000", "123.321, 123456789.987654321, 123456789, 123.321"))) {
+            assertThat(query("SELECT min(short_decimal), min(long_decimal), min(a_bigint), min(t_double) FROM " + testTable.getName())).isFullyPushedDown();
+            assertThat(query("SELECT max(short_decimal), max(long_decimal), max(a_bigint), max(t_double) FROM " + testTable.getName())).isFullyPushedDown();
+            assertThat(query("SELECT sum(short_decimal), sum(long_decimal), sum(a_bigint), sum(t_double) FROM " + testTable.getName())).isFullyPushedDown();
+            assertThat(query("SELECT avg(short_decimal), avg(long_decimal), avg(a_bigint), avg(t_double) FROM " + testTable.getName())).isFullyPushedDown();
         }
+    }
 
-        // TODO array_agg returns array, so it could be supported
-        assertThat(query("SELECT array_agg(nationkey) FROM nation")).isNotFullyPushedDown(AggregationNode.class);
-
-        // histogram returns map, which is not supported
-        assertThat(query("SELECT histogram(regionkey) FROM nation")).isNotFullyPushedDown(AggregationNode.class);
-
-        // multimap_agg returns multimap, which is not supported
-        assertThat(query("SELECT multimap_agg(regionkey, nationkey) FROM nation")).isNotFullyPushedDown(AggregationNode.class);
+    @Override
+    protected TestTable createTableWithNumericTypes(String name, List<String> rows)
```

Comment:
"with decimal"?
"with short and long decimals"?

Reply:
There are also bigints and doubles. It's used for testing `count`, `min`, `max`, `sum`, and `avg`.

`simpleAggregationTestTable`?